pkgs/top-level/all-packages.nix

```diff
@@ -13055,6 +13055,8 @@ with pkgs;
 
   typos = callPackage ../development/tools/typos { };
 
+  typst = callPackage ../tools/typesetting/typst { };
```

Comment:
Should this be named `typst-cli` instead? What do you think? I prefer it being named `typst-cli` while being in the `typst` directory, perhaps they will release a gui app in the future (or not), then it could be placed in that dir.

Reply:
I prefer `typst` since this is the name of the executable and the repository, if the repo gets a gui, it can go in the same package so we can save compilation time, or it can go to a separate package with the name of the gui executable